plugins/view-transitions/includes/class-plvt-view-transition-animation.php

```diff
@@ -0,0 +1,283 @@
+<?php
+/**
+ * Class 'PLVT_View_Transition_Animation'.
+ *
+ * @package view-transitions
+ * @since 1.0.0
+ */
+
+/**
+ * Class representing a view transition animation.
+ *
+ * @since 1.0.0
+ * @access private
+ */
+final class PLVT_View_Transition_Animation {
+
+	/**
+	 * The unique animation slug.
+	 *
+	 * @since 1.0.0
+	 * @var string
+	 */
+	private $slug;
+
+	/**
+	 * Unique aliases for the animation, if any.
+	 *
+	 * @since 1.0.0
+	 * @var string[]
+	 */
+	private $aliases = array();
+
+	/**
+	 * Whether the animation uses a stylesheet.
+	 *
+	 * If so, the stylesheet will be `/wp-includes/css/view-transitions-animation-{$slug}.css`.
```

Comment:
I think in the context of the current feature plugin the path is supposed to be like this:
```suggestion
	 * If so, the stylesheet will be `/css/view-transitions-animation-{$slug}.css`.
```

Also, I just wanted to know if the prefix used in CSS is intentionally `wp-*`, as I was thinking that in the context of the current feature plugin, it should be `plvt-*`.

Reply:
Great points! Will update those.

Reply:
Fixed in https://github.com/WordPress/performance/pull/2016/commits/8adf5fec4884e292dfa41e1cfcad6006c3be7478